lib/src/test/kotlin/starknet/account/StandardAccountTest.kt

```diff
@@ -80,45 +82,176 @@ class StandardAccountTest {
 
     @ParameterizedTest
     @MethodSource("getAccounts")
-    fun `sign single call test`(account: Account) {
+    fun `constructor signer creation`(accountAndProvider: AccountAndProvider) {
+        val (_, provider) = accountAndProvider
+        val privateKey = Felt(1234)
+        val account = StandardAccount(provider, Felt.ZERO, privateKey)
+
+        assertNotNull(account.signer)
+
+        val privateKeyField = account.signer.javaClass.getDeclaredField("privateKey")
+        privateKeyField.isAccessible = true
+        val signerPrivateKey = privateKeyField.get(account.signer) as Felt
+        assertEquals(privateKey, signerPrivateKey)
+    }
```

Comment:
This is not needed at all. We can just test if it works properly with the default constructor. No need for hacks like that.